src/engine/include/includejs/engine_value.h

```diff
@@ -12,6 +12,19 @@
 
 namespace includejs {
 
+enum class JSValueType {
```

Comment:
Can we call it just `ValueType`? The `JS` prefix is redundant, as this is already `IncludeJS`

Reply:
yeah for sure :)